src/screens/Admin/screens/InstructorPulse/index.js

```diff
@@ -10,13 +12,15 @@ export default ({instructors}) => (
     main={
       <InstructorsList
         title={unpublishedTitleText}
-        instructors={filter(instructors, ['published_lessons', 0])}
+        instructors={filter(instructors, ['published_lessons', 0])
+          .sort(sortBy('-submitted_lessons', '-claimed_lessons', '-approved_lessons', '-id'))}
       />
     }
     aside={
       <InstructorsList
         title={publishedTitleText}
-        instructors={reject(instructors, ['published_lessons', 0])}
+        instructors={reject(instructors, ['published_lessons', 0])
+          .sort(sortBy('-submitted_lessons', '-claimed_lessons', 'published_lessons', '-approved_lessons', '-id'))}
```

Comment:
Published lesson count is the only one that sorts ascending, so people with the *least* number of published lessons are closer to the top of the list. Basically, I will **always** expect @johnlindquist to be at the bottom of this list. He needs the least help.